index.js

```diff
@@ -183,6 +183,6 @@ module.exports.expandInputs = expandInputs;
 module.exports.formats = {
   html: require('./lib/output/html'),
   md: require('./lib/output/markdown'),
-  mdast: require('./lib/output/markdown_ast'),
+  remark: require('./lib/output/markdown_ast'),
```

Comment:
Good point, that’s what I do internally indeed, but I’m not sure whether you’d want to go with the (in the future) better known `remark`, or with the name of the AST.

I can amend if need-be!